static/app/views/dashboards/datasetConfig/releases.tsx

```diff
@@ -291,6 +291,7 @@ export function transformSessionsResponseToTable(
   const singleRow = rows[0];
   const meta = {
     ...changeObjectValuesToTypes(omit(singleRow, 'id')),
+    fields: changeObjectValuesToTypes(omit(singleRow, 'id')),
```

Comment:
I wanted to completely replace the `...changeObjectValuesToTypes(omit(singleRow, 'id')),` but I did a quick grep for how we access `tableMeta` and it seems like the response type usually has all of the fields with their meta info, and then we pull it into `fields` like this, so I'm just going for consistency here to fix the original ask.

We should migrate and ensure consistency of this data type so we never have to consider `Object.keys(tableMeta)` or `Object.keys(tableMeta.fields)` again, but I think that's for another PR